lobby/src/test/java/org/triplea/lobby/server/login/forgot/password/create/PasswordGeneratorTest.java

```diff
@@ -0,0 +1,33 @@
+package org.triplea.lobby.server.login.forgot.password.create;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+class PasswordGeneratorTest {
+
+  @Test
+  void verifyPasswordLength() {
+    assertThat(
+        new PasswordGenerator().generatePassword().length(), is(PasswordGenerator.PASSWORD_LENGTH));
+  }
+
+  /**
+   * Generate a password, check we do not have it in a set, add the password to the set and repeat
+   * many times.
+   */
+  @Test
+  void verifyPasswordChanges() {
+    final var passwordGenerator = new PasswordGenerator();
+
+    final Set<String> strings = new HashSet<>();
+    for (int i = 0; i < 1000; i++) {
+      final String generated = passwordGenerator.generatePassword();
+      assertThat(strings.contains(generated), is(false));
+      strings.add(generated);
```

Comment:
Imagine this test failing once in the next 100 years or something ^^

Reply:
I suspect probably even more rare than 100 years. If it were to fail, I'd guess a dev probably would ignore it.. Ideally not test failure is ignored, they all should have some work put in to make them deterministic or fixed otherwise. 

There is non-determinism in this test, but it's almost a good thing as we increase test coverage each time we run the test. I added a note recommending how to correct the code if we see a failure: e182b430a